BrainPortal/lib/exception_helpers.rb

```diff
@@ -34,40 +36,59 @@ def self.included(includer) #:nodoc:
     end
   end
 
+  NOT_FOUND_MSG = "The object you requested does not exist or is not accessible to you."
```

Comment:
Constants should be defined at the top of the file, after Revision_info, separated by one blank line before and after the constants (and not in between). If no documentation is provided, or they must not be documented, append #:nodoc: . Look at `scir.rb` for examples.